Sources/SwiftkubeClient/Config/KubernetesClientConfig.swift

```diff
@@ -228,3 +228,51 @@ private extension AuthInfo {
 		return nil
 	}
 }
+
+public extension KubernetesClientConfig {
```

Comment:
Can you make this implement the `KubernetesClientConfigLoader` protocol analogous to the other loaders. It would however mean that we would have to create an instance of the struct and, as of now, it is a one-shot-then-throw-away instance (as you can see [here](https://github.com/swiftkube/client/blob/f40782168edb0f466448b744df4a51333a4c05d1/Sources/SwiftkubeClient/Client/KubernetesClient.swift#L103)).

Reply:
Ah yes, I did actually consider this when I first looked at it, but the one-shot-then-throw-away thing confused me. But yeah, if nothing else it keeps everything consistent and maintainable. I will change it